src/components/Counter.component.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react'
+// TODO: uncomment this for Part 1
+// import PropTypes from 'prop-types'
+import './../styles/instructions.css'
+
+class Counter extends Component {
+  render() {
+    var count = this.props.count
+    var incrementCount = function() {
+      count++
+      console.log(count)
+    }
+    var decrementCount = function() {
+      count--
+      console.log(count)
+    }
```

Comment:
if you want to update a value in the app, you should use `state` and follow that flow instead of locally changing a variable. you _especially_ shouldn't be changing these inside the `render` function itself, it breaks the paradigm of a component's `render()` being a pure function of its `state` and `props`.

Reply:
This is why part 3 doesn't work properly. You should have functions that use `setState` and display the `count` value as the component's `state` or `props`